Synthesis failed for nodejs-automl

Hello! Autosynth couldn't regenerate nodejs-automl. :broken_heart:

Here's the output from running `synth.py`:

```
05de3e1e14a0b07eab8b474e669164dbd31f81fb
2020-09-11 03:17:04,362 autosynth [DEBUG] > Running: git log -1 --pretty=%at 968465a1cad496e1292ef4584a054a35f756ff94
2020-09-11 03:17:04,366 autosynth [DEBUG] > Running: git log -1 --pretty=%at a9eea2c50b7dce0dffcc010c1caf712802155403
2020-09-11 03:17:04,369 autosynth [DEBUG] > Running: git log -1 --pretty=%at 637f8aa1373a0a6be1b626a282f2e13d7d6d7d6c
2020-09-11 03:17:04,372 autosynth [DEBUG] > Running: git log -1 --pretty=%at d0198121927f606e113275a4b0f3560a7a821470
2020-09-11 03:17:04,375 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8cf6d2834ad14318e64429c3b94f6443ae83daf9
2020-09-11 03:17:04,378 autosynth [DEBUG] > Running: git log -1 --pretty=%at 019c7168faa0e56619f792693a8acdb30d6de19b
2020-09-11 03:17:04,382 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5d916ec54cadd4674e80e6555d0c6a78849ef4a7
2020-09-11 03:17:04,385 autosynth [DEBUG] > Running: git log -1 --pretty=%at cbcd64279572769b4d350bf8078bcd1f151c9684
2020-09-11 03:17:04,388 autosynth [DEBUG] > Running: git log -1 --pretty=%at 80f46100c047bc47efe0025ee537dc8ee413ad04
2020-09-11 03:17:04,391 autosynth [DEBUG] > Running: git log -1 --pretty=%at d91dd8aac77f7a9c5506c238038a26fa4f9e361e
2020-09-11 03:17:04,394 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7b3766035373f5da99abf9413f1f7c29f108a2fd
2020-09-11 03:17:04,397 autosynth [DEBUG] > Running: git log -1 --pretty=%at 32c758f11b8c578f515a746c9d263b82a615a77c
2020-09-11 03:17:04,400 autosynth [DEBUG] > Running: git log -1 --pretty=%at b2c32f1c8a4094f0f47fcf5d10f0b6f2bfb3387d
2020-09-11 03:17:04,403 autosynth [DEBUG] > Running: git log -1 --pretty=%at 470e98896484cdce77e073e83a2c4f9157770f8c
2020-09-11 03:17:04,407 autosynth [DEBUG] > Running: git log -1 --pretty=%at d302f93d7f47e2852e585ac35ab2d15585717ec0
2020-09-11 03:17:04,410 autosynth [DEBUG] > Running: git log -1 --pretty=%at ffcee7952b74f647cbb3ef021d95422f10816fca
2020-09-11 03:17:04,413 autosynth [DEBUG] > Running: git log -1 --pretty=%at b15c0c042cdea746fc19856527d8baf947c3c220
2020-09-11 03:17:04,417 autosynth [DEBUG] > Running: git checkout bc929c069d2645df2026f4ff369795ba6118ca1f
Note: checking out 'bc929c069d2645df2026f4ff369795ba6118ca1f'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at bc929c0 samples: fix line endings (#443)
2020-09-11 03:17:04,491 autosynth [DEBUG] > Running: git checkout b15c0c042cdea746fc19856527d8baf947c3c220
Note: checking out 'b15c0c042cdea746fc19856527d8baf947c3c220'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at b15c0c0 fix: don't attempt to create redundant PR to start tracking obsolete files (#755)
2020-09-11 03:17:04,504 autosynth [DEBUG] > Running: git branch -f autosynth-76
2020-09-11 03:17:04,507 autosynth [DEBUG] > Running: git checkout autosynth-76
Switched to branch 'autosynth-76'
2020-09-11 03:17:04,512 autosynth [INFO] > Running synthtool
2020-09-11 03:17:04,512 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2020-09-11 03:17:04,513 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/nodejs-automl/76/sponge_log.log
2020-09-11 03:17:04,514 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2020-09-11 03:17:04,737 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-automl/synth.py.
On branch autosynth-76
nothing to commit, working tree clean
2020-09-11 03:17:04,901 synthtool [DEBUG] > Ensuring dependencies.
DEBUG:synthtool:Ensuring dependencies.
2020-09-11 03:17:04,911 synthtool [DEBUG] > Cloning googleapis.
DEBUG:synthtool:Cloning googleapis.
2020-09-11 03:17:07,067 synthtool [DEBUG] > Generating code for: //google/cloud/automl/v1:automl-v1-nodejs.
DEBUG:synthtool:Generating code for: //google/cloud/automl/v1:automl-v1-nodejs.
2020-09-11 03:20:11,706 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmp0vvnqfhi.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmp0vvnqfhi.
2020-09-11 03:20:11,766 synthtool [DEBUG] > Generating code for: //google/cloud/automl/v1beta1:automl-v1beta1-nodejs.
DEBUG:synthtool:Generating code for: //google/cloud/automl/v1beta1:automl-v1beta1-nodejs.
2020-09-11 03:20:17,983 synthtool [SUCCESS] > Generated code into /tmpfs/tmp/tmpqbsdcbh5.
SUCCESS:synthtool:Generated code into /tmpfs/tmp/tmpqbsdcbh5.
2020-09-11 03:20:18,060 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
DEBUG:synthtool:Using precloned repo /home/kbuilder/.cache/synthtool/synthtool
2020-09-11 03:20:18,080 synthtool [INFO] > successfully generate `src/index.ts`
INFO:synthtool:successfully generate `src/index.ts`
.eslintignore
.eslintrc.json
.gitattributes
.github/ISSUE_TEMPLATE/bug_report.md
.github/ISSUE_TEMPLATE/feature_request.md
.github/ISSUE_TEMPLATE/support_request.md
.github/PULL_REQUEST_TEMPLATE.md
.github/release-please.yml
.github/workflows/ci.yaml
.kokoro/.gitattributes
.kokoro/common.cfg
.kokoro/continuous/node10/common.cfg
.kokoro/continuous/node10/docs.cfg
.kokoro/continuous/node10/test.cfg
.kokoro/continuous/node12/common.cfg
.kokoro/continuous/node12/lint.cfg
.kokoro/continuous/node12/samples-test.cfg
.kokoro/continuous/node12/system-test.cfg
.kokoro/continuous/node12/test.cfg
.kokoro/docs.sh
.kokoro/lint.sh
.kokoro/populate-secrets.sh
.kokoro/presubmit/node10/common.cfg
.kokoro/presubmit/node12/common.cfg
.kokoro/presubmit/node12/samples-test.cfg
.kokoro/presubmit/node12/system-test.cfg
.kokoro/presubmit/node12/test.cfg
.kokoro/publish.sh
.kokoro/release/docs-devsite.cfg
.kokoro/release/docs-devsite.sh
.kokoro/release/docs.cfg
.kokoro/release/docs.sh
.kokoro/release/publish.cfg
.kokoro/samples-test.sh
.kokoro/system-test.sh
.kokoro/test.bat
.kokoro/test.sh
.kokoro/trampoline.sh
.mocharc.js
.nycrc
.prettierignore
.prettierrc.js
CODE_OF_CONDUCT.md
CONTRIBUTING.md
LICENSE
README.md
api-extractor.json
renovate.json
samples/README.md
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-automl/synth.py", line 36, in <module>
    node.postprocess_gapic_library(excludes=['.kokoro/trampoline.sh'])
TypeError: postprocess_gapic_library() got an unexpected keyword argument 'excludes'
2020-09-11 03:20:18,304 autosynth [ERROR] > Synthesis failed
2020-09-11 03:20:18,305 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at bc929c0 samples: fix line endings (#443)
2020-09-11 03:20:18,322 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-09-11 03:20:18,335 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing api-extractor.json
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 692, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 541, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 672, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/c7c3a667-acb8-4d7d-99e7-4b2c2ae42009/targets/github%2Fsynthtool;config=default/tests;query=nodejs-automl;failed=false).

--- a/synth.py
+++ b/synth.py
@@ -31,6 +31,6 @@ for version in versions:
 common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library(
     source_location='build/src', versions=versions, default_version='v1')
-s.copy(templates)
+s.copy(templates, excludes=['.kokoro/trampoline.sh'])
 
-node.postprocess_gapic_library(excludes=['.kokoro/trampoline.sh'])
+node.postprocess_gapic_library()

